.github/CODEOWNERS

```diff
@@ -48,6 +48,7 @@
 # Testing
 /tests/helpers/boring_model.py          @williamfalcon @tchaton @borda
 
-/.github/CODEOWNERS     @williamfalcon
-/README.md              @williamfalcon @edenlightning @borda
-/setup.py               @williamfalcon @borda
+/.github/CODEOWNERS         @williamfalcon
```

Comment:
Can you add Carlos and I for the profilers ?

Reply:
And @kaushikb11 for TPU Plugins.

Reply:
you are already there for profilers :]
@kaushikb11 mind suggest yourself where to put you

Reply:
And other places as well. Would like a notification to do reviews :)

Reply:
@kaushikb11 for general notifications https://github.com/PyTorchLightning/pytorch-lightning/pull/6885/files#r609448605
no pls suggest the specific as code-owner

Reply:
I have pushed the suggestions!